python3/vim_pandoc/bib/collator.py

```diff
@@ -1,6 +1,7 @@
 import os
 import vim
 from glob import glob
+from itertools import chain
```

Comment:
Does this extra dependency need to be documented anywhere?

Reply:
I wouldn't think so. `itertools` is part of the python standard library, just like `os`, `glob`, and `subprocess`.

Hpwever, it's easy to rewrite the code to get rid of this dependency.